examples/streaming/kafka-hdfs-pipeline/src/main/scala/org/apache/gearpump/examples/streaming/pipeline/KafkaHdfsPipeLine.scala

```diff
@@ -0,0 +1,150 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.examples.streaming.pipeline
+
+import com.julianpeeters.avro.annotations._
+import com.typesafe.config.ConfigFactory
+import org.apache.gearpump._
+import org.apache.gearpump.cluster.UserConfig
+import org.apache.gearpump.cluster.client.ClientContext
+import org.apache.gearpump.cluster.main.{ArgumentsParser, CLIOption, ParseResult}
+import org.apache.gearpump.partitioner.ShufflePartitioner
+import org.apache.gearpump.streaming.kafka.KafkaSource
+import org.apache.gearpump.streaming.kafka.lib.KafkaConfig
+import KafkaConfig._
+import org.apache.gearpump.streaming.source.DataSourceProcessor
+import org.apache.gearpump.streaming.task.{StartTime, Task, TaskContext}
+import org.apache.gearpump.streaming.transaction.api.TimeReplayableSource
+import org.apache.gearpump.streaming.{Processor, StreamApplication}
+import org.apache.gearpump.util.Graph._
+import org.apache.gearpump.util.{Constants, Graph, LogUtil}
+import org.slf4j.Logger
+
+import scala.util.{Failure, Success, Try}
+
+case class SpaceShuttleMessage(id: String, on: String, body: String)
+
+/**
+ * This annotation will translate the case class into a Avro type class.
+ * Fields must be vars in order to be compatible with the SpecificRecord API.
+ * We will use this case class directly in ParquetWriterTask.
+ */
+@AvroRecord
+case class SpaceShuttleRecord(var timestamp: Double, var vectorClass: Double, var count: Double)
+
+
+
+
+object KafkaHdfsPipeLine extends App with ArgumentsParser {
+  private val LOG: Logger = LogUtil.getLogger(getClass)
+
+  override val options: Array[(String, CLIOption[Any])] = Array(
+    "reader"-> CLIOption[Int]("<kafka data reader number>", required = false, defaultValue = Some(2)),
+    "scorer"-> CLIOption[Int]("<scorer number>", required = false, defaultValue = Some(2)),
+    "writer"-> CLIOption[Int]("<parquet file writer number>", required = false, defaultValue = Some(1)),
+    "output"-> CLIOption[String]("<output path directory>", required = false, defaultValue = Some("file:///tmp/parquet")),
+    "conf" -> CLIOption[String]("<conf file>", required = false, defaultValue = Some("conf/kafka-hdfs-pipeline.conf"))
+  )
+
+  val context = ClientContext()
+  implicit val system = context.system
+
+  def application(context: ClientContext, config: ParseResult): Unit = {
+    val readerNum = config.getInt("reader")
+    val scorerNum = config.getInt("scorer")
+    val writerNum = config.getInt("writer")
+    val outputPath = config.getString("output")
+    val pipeLinePath = config.getString("conf")
+    val file = new java.io.File(pipeLinePath)
+    val pipeLineConfig = file.exists() match {
+      case true =>
+        ConfigFactory.parseFile(file)
+      case false =>
+        val fallback =
```

Comment:
no need for the long list of kafka configs now. `val source = new KafkaSource("topic-105", "10.10.10.46:2181,10.10.10.236:2181,10.10.10.164:2181/kafka")` will do